libs/sdk-ui-kit/src/List/tests/LegacyMultiSelectList.test.tsx

```diff
@@ -97,179 +101,173 @@ describe("LegacyMultiSelectList", () => {
             });
-            clickCheckbox(wrapper);
+            await userEvent.click(screen.getAllByRole("checkbox")[0]);
             expect(onSelectAll).toHaveBeenCalledTimes(1);
         });
 
         it("should have selectAll enabled if not searching", () => {
-            const wrapper = renderList({
+            renderList({
                 items,
                 selection: [firstItem],
                 selectAllCheckbox: true,
                 isSearching: false,
             });
-            expect(wrapper.find("label.s-select-all-checkbox")).toHaveLength(1);
-            expect(wrapper.find("label.s-select-all-checkbox").hasClass("disabled")).toBeFalsy();
+
+            expect(document.querySelector(".s-select-all-checkbox")).not.toHaveClass("disabled");
         });
 
         describe("selection footer info", () => {
             it("should show all when nothing is selected", () => {
-                const wrapper = renderList({
+                renderList({
                     items,
                     selection: [],
                     selectAllCheckbox: true,
                     isInverted: true,
                     isSearching: false,
                     tagName: "Attribute",
                 });
-                expect(wrapper.find(".s-list-status-bar")).toHaveLength(1);
-                expect(wrapper.find(".s-list-status-bar").text().replace(/\xA0/g, " ")).toBe(
+
+                expect(document.querySelector(".s-list-status-bar").textContent.replace(/\xA0/g, " ")).toBe(
                     "Attribute is All",
                 );
             });
 
             it("should show none when nothing is selected", () => {
-                const wrapper = renderList({
+                renderList({
                     items,
                     selection: [],
                     selectAllCheckbox: true,
                     isSearching: false,
                     tagName: "Attribute",
                 });
-                expect(wrapper.find(".s-list-status-bar")).toHaveLength(1);
-                expect(wrapper.find(".s-list-status-bar").text().replace(/\xA0/g, " ")).toBe(
+                expect(document.querySelector(".s-list-status-bar").textContent.replace(/\xA0/g, " ")).toBe(
                     "Attribute is (None)",
                 );
             });
 
             it("should show selected item info", () => {
-                const wrapper = renderList({
+                renderList({
                     items,
                     selection: [items[0]],
                     selectAllCheckbox: true,
                     isSearching: false,
                     tagName: "Attribute",
                 });
-                expect(wrapper.find(".s-list-status-bar")).toHaveLength(1);
-                expect(wrapper.find(".s-list-status-bar").text().replace(/\xA0/g, " ")).toBe(
+
+                expect(document.querySelector(".s-list-status-bar").textContent.replace(/\xA0/g, " ")).toBe(
```

Comment:
Maybe we could add aria label `list-status-bar` instead of using class name for selection

Reply:
Added role `list-status-bar`